README.md

```diff
@@ -40,141 +40,141 @@ npm run lint-fix # format source files
+| [#112][#112] | [SHOULD used open-ended list of values (`x-extensible-enum`) for enumerations][#112] | :heavy_check_mark: | - |
+| [#113][#113] | [SHOULD avoid versioning][#113] | - | B |
+| [#114][#114] | [MUST use media type versioning][#114] | - | - |
+| [#115][#115] | [MUST not use URI versioning][#115] | - | A |
+| [#116][#116] | [MUST use semantic versioning][#116] | :heavy_check_mark: | B |
+| [#118][#118] | [MUST property names must be ASCII snake_case (and never camelCase): `^[a-z_][a-z_0-9]*$`][#118] | :heavy_check_mark: | A |
+| [#120][#120] | [SHOULD pluralize array names][#120] | - | B |
+| [#122][#122] | [MUST not use `null` for boolean properties][#122] | - | - |
+| [#123][#123] | [MUST use same semantics for `null` and absent properties][#123] | - | - |
+| [#124][#124] | [SHOULD not use `null` for empty arrays][#124] | - | - |
+| [#125][#125] | [SHOULD represent enumerations as strings][#125] | - | - |
+| [#126][#126] | [SHOULD define dates properties compliant with RFC 3339][#126] | - | - |
+| [#127][#127] | [SHOULD define time durations and intervals properties conform to ISO 8601][#127] | - | - |
+| [#128][#128] | [SHOULD use standards for country, language and currency codes][#128] | - | - |
+| [#129][#129] | [MUST use lowercase separate words with hyphens for path segments][#129] | - | - |
+| [#130][#130] | [MUST use snake_case (never camelCase) for query parameters][#130] | - | B |
+| [#132][#132] | [SHOULD prefer hyphenated-pascal-case for HTTP header fields][#132] | - | - |
+| [#133][#133] | [MAY use standardized headers][#133] | - | - |
+| [#134][#134] | [MUST pluralize resource names][#134] | - | A |
+| [#135][#135] | [SHOULD not use /api as base path][#135] | - | ? |
+| [#136][#136] | [MUST use normalized paths without empty path segments and trailing slashes][#136] | - | - |
+| [#137][#137] | [MUST stick to conventional query parameters][#137] | - | - |
+| [#138][#138] | [MUST avoid actions — think about resources][#138] | - | A |
+| [#139][#139] | [SHOULD model complete business processes][#139] | - | - |
+| [#140][#140] | [SHOULD define _useful_ resources][#140] | - | - |
+| [#141][#141] | [MUST keep URLs verb-free][#141] | - | A |
+| [#142][#142] | [MUST use domain-specific resource names][#142] | - | - |
+| [#143][#143] | [MUST identify resources and sub-resources via path segments][#143] | - | - |
+| [#144][#144] | [SHOULD only use UUIDs if necessary][#144] | - | - |
+| [#145][#145] | [MAY consider using (non-)nested URLs][#145] | - | - |
+| [#146][#146] | [SHOULD limit number of resource types][#146] | - | ? |
+| [#147][#147] | [SHOULD limit number of sub-resource levels][#147] | - | B |
+| [#148][#148] | [MUST use HTTP methods correctly][#148] | - | A |
+| [#149][#149] | [MUST fulfill common method properties][#149] | - | - |
+| [#150][#150] | [MUST use standard HTTP status codes][#150] | - | - |
+| [#151][#151] | [MUST specify success and error responses][#151] | - | - |
```

Comment:
```suggestion
| [#151][#151] | [MUST specify success and error responses][#151] | - | B |
```
otherwise we'll end up with api specs without error definitions